src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/parenpad/Example1.java

```diff
@@ -0,0 +1,31 @@
+/*
+ParenPad
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.parenpad;
+
+class Example1 {
```

Comment:
Was leaving out xdoc comments intentional?

Reply:
So you mean  missed `// xdoc section -- start` ?

Probably not intentional, but still can be done on the moment we will make them actionable.

Reply:
Yes. I completely missed it. Added back. Done.

Reply:
@stoyanK7 , do we have this in ideas for validation by checkstyle? Please add to that ticket if missed. I think all of them will have such comments.

Reply:
@romani added to https://github.com/checkstyle/checkstyle/issues/13244#issue-1761844973

Reply:
We already have validation for that. It got merged a few hours ago.

https://github.com/checkstyle/checkstyle/blob/8057f64b06848ced41ff3d53438d184999e8c94a/config/checkstyle-examples-checks.xml#L47-L60